app/controllers/mail/regions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Mail::RegionsController < Mail::ApplicationController
+
+  before_action :fetch_region
+
+  def summary
+    summary_period = PlaceMailer::SUMMARY_PERIOD
+    region_label = RegionDecorator.get_name(@place.province_code, @place.country.country_code)
+    @subject = I18n.translate('mail.place.summary.subject', place: region_label)
+
+    @start_of_period = summary_period.ago.beginning_of_week
+    @end_of_period = @start_of_period + summary_period
+    @end_of_period = Time.now # For testing
+    query = ['events.created_at >= ? AND events.created_at <= ?', @start_of_period, @end_of_period]
+
+    @new_events = @place.events.publicly_visible.where(*query)
```

Comment:
Just in case you hadn't seen it before, you can use ranges for these kinds of queries, eg:

```suggestion
    @new_events = @place.events.publicly_visible.where(created_at: @start_of_period..@end_of_period)
```

Reply:
I did not know. Good to know, thanks

Reply:
You might want to grep the codebase for eg. `>= ?`. Seems like there a few more occurrences within and outside of this PR.